virtcontainers/kata_agent.go

```diff
@@ -399,6 +399,43 @@ func (k *kataAgent) internalConfigure(h hypervisor, id string, builtin bool, con
 	return nil
 }
 
+func setupLoggingBindMount(id string) error {
+	// Create path for observability logs:
+	path := filepath.Join("/var/log/kata", id)
```

Comment:
The static string `/var/log/kata` is used twice so please create a `const` for it.